### PR TITLE
Twnia2/user guide] 1. 可用計算資源

### PR DESCRIPTION
**Pull request by ting88510**
#211

### CODE REVIEW

File: TWNIA2/1. User Guides/1. 可用計算資源.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+tags: Guide, TWNIA2, EN
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# Available computing resources
+
+Taiwania2 is an AI supercomputer mainframe using 2,016  NVIDIA® Tesla® V100 GPU, with 9 PFLOPS(9 peta floating-point operations per second) performance.
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/1. 可用計算資源.md:12]: can we delete this paragraph?

File: TWNIA2/1. User Guides/1. 可用計算資源.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+As mentioned in Service overview, Taiwania2 (HPC CLI)  shares **252 computing nodes** and **2,016 NVIDIA® Tesla® V100 GPUs** with some TWCC computing services, provides **9 PFLOPS** (9 peta floating-point operations per second) excellent performance, supports up to 300 GB/s transfer rate between GPUs with NVLINK, also, provides a high-bandwidth, low-latency parallel computing network.
```
Comment (VigaWei) [inline TWNIA2/1. User Guides/1. 可用計算資源.md:14]: As mentioned in the Service overview, Taiwania2 (HPC CLI)  shares **252 computing nodes** and **2,016 NVIDIA® Tesla® V100 GPUs** with some TWCC computing services. Taiwania 2 has excellent performance of **9 PFLOPS** (9 Peta floating-point operations per second), supports up to 300 GB/s transfer rate between GPUs with NVLINK, also provides a high-bandwidth, low-latency parallel computing network.

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
Revised, thanks!!

**Pull request merged**